### ISSUE TITLE
Bug: mask doesn't pass exit code

### ISSUE THREAD

**Issue by DrSensor** [jakedeichert/mask#24]
Just noticed that `mask` command doesn't forward the exit code. This make this kind of task impossible:

~~~markdown
## start
> Run project

```sh
mask check && mask compile && \
node dist/index.js
# this ☝️ line will always be executed regardless the compile & check process fail or success
# it should not be executed when check or compile process fail
```
~~~

**Comment by jakedeichert** [comment]
Actually, this is done but unreleased, see the [changelog](https://github.com/jakedeichert/mask/blob/master/CHANGELOG.md#unreleased).
> Propagate exit status of child process to main process

I'll release 0.4.0 tonight likely!

**Comment by jakedeichert** [comment]
Released [v0.4.0](https://github.com/jakedeichert/mask/releases/tag/v0.4.0)
Give it a try 👍
